src/term.rs

```diff
@@ -19,6 +19,19 @@ pub enum Term<VS: VersionSet> {
     Negative(VS),
 }
 
+impl<VS: VersionSet> PartialEq for Term<VS> {
```

Comment:
I think this is currently wrong on dev?

Reply:
Thank you for opening a [issue](https://github.com/pubgrub-rs/pubgrub/issues/178) to discuss this.